src/wopiutils.py

```diff
@@ -122,11 +124,10 @@ def getLockName(filename):
 
 def retrieveWopiLock(fileid, operation, lock, acctok):
   '''Retrieves and logs an existing lock for a given file'''
-  l = b''
   encacctok = flask.request.args['access_token'][-20:] if 'access_token' in flask.request.args else 'N/A'
-  for line in _ctx['st'].readfile(acctok['endpoint'], getLockName(acctok['filename']),
-                                  _ctx['wopi'].lockruid, _ctx['wopi'].lockrgid):
-    if 'ERROR on read' in str(line):
+  l = b''
```

Comment:
What is this l?

Reply:
short for `lock`, it is actually the content of the lock file being built in the next line. Maybe rename to `lockcontent`?

Reply:
I guess it's the same at line 167?

Reply:
Yep, the code was pretty much copy-pasted

Reply:
✅ Will update there as well then